src/kubernetes-ingress-controller/concepts/gateway-api.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Gateway API
+alpha: true
+---
+
+[Gateway API](https://gateway-api.sigs.k8s.io/) is a set of resources for
+configuring networking in Kubernetes. It expands on Ingress to configure
+additional types of routes (TCP, UDP, and TLS in addition to HTTP/HTTPS),
+support backends other than Service, and manage the proxies that implement
+routes.
+
+Gateway API and Kong's implementation of Gateway API are both in alpha stage and
```

Comment:
Gateway API itself is in beta not alpha, right? Even though their README states `alpha`: https://github.com/kubernetes-sigs/gateway-api/#status (I believe we can contribute a change to update that)

Reply:
Oh, yeah please do update that README.md that was just an oversight. :facepalm: 

Some APIs are in beta: `GatewayClass`, `Gateway`, `HTTPRoute` and the others are still in alpha so it's more accurate for us to say "we have beta level support for `v1beta1.GatewayClass`, `v1beta1.Gateway`, and `v1beta1.HTTPRoute` and for all other APIs we provide alpha level support".

Beta level support can be defined as we're committed to the feature going forward, and it's expected to work according to "core" requirements of the upstream API. Use in production at your own risk.

Alpha level support can be defined as "preview": we're not committed to any of it, but we welcome feedback. **Do Not** use in production.

Reply:
Added a PR for this: https://github.com/kubernetes-sigs/gateway-api/pull/1322

Reply:
Do we want to just remove mention of upstream? Something like "Our implementation is alpha, but upstream is beta, except for the parts that are still in alpha" sounds awkward, and I think just mentioning that our implementation is alpha is sufficient for what readers need to know about its production-readiness. If we do want to include both, what _would_ be a non-awkward phrasing?

Reply:
Our implementation is _beta_ however. I think the distinction that we have some APIs and relevant features that we consider at a beta level (both in kong, and in terms of the API version itself) vs the alpha APIs.

Reply:
Updated to use generic language that just indicates it's not stable yet.